lib/install.js

```diff
@@ -242,8 +242,15 @@ Installer.prototype.run = function (_cb) {
     }
   }
 
+  var preinstallSteps = []
   var installSteps = []
   var postInstallSteps = []
+
+  preinstallSteps.push(
```

Comment:
Chains are run in sequence and stop after the first error. The only reason to have multiple chains is if you want to run a later chain even though an earlier one failed.